rush-plugins/rush-amazon-s3-build-cache-plugin/src/AmazonS3BuildCacheProvider.ts

```diff
@@ -7,7 +7,7 @@ import type {
   ICredentialCacheEntry,
   ICredentialCache,
   RushSession
-} from '@microsoft/rush-lib';
```

Comment:
You should be able to remove this dependency.

Reply:
Fixed